API/ev3.c

```diff
@@ -24,44 +24,22 @@ int __attribute__((constructor)) InitEV3 (void)
 	    return 1;
 
 	OutputInit();
-	InitSensors();
+	SensorsInit();
 	ButtonLedInit();
 	LcdInit();
-	LcdClean();
 	SoundInit();
-	initialized = true;
-
-	return 1;
-}
-
-
-int CloseEV3()

```

Comment:
What if existing code already uses those? How about making `CloseEV3` and `FreeEV3` aliases for `ExitEV3` and make successive calls no-ops?

Reply:
Calling only `CloseEV3` or `ExitEV3` would not properly shutdown the EV3 and calling both in the right order would be equivalent to `FreeEV3`. That's why I removed them.
Successive calls to the actual `Exit*` functions that are used in `ExitEV3` are no-ops already. 
Good point on compatibility though. I'll make the alias definitions.

Reply:
`CloseEV3` and `ExitEV3` now call `FreeEV3`